heft-plugins/heft-dev-cert-plugin/package.json

```diff
@@ -14,14 +14,16 @@
   "scripts": {
     "build": "heft build --clean",
     "start": "heft test --clean --watch",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean"
   },
   "peerDependencies": {
     "@rushstack/heft": "^0.45.14"
   },
   "dependencies": {
     "@rushstack/debug-certificate-manager": "workspace:*",
+    "@rushstack/heft-webpack4-plugin": "workspace:*",
```

Comment:
Since the webpack 4 plugin and the webpack 5 plugin both declare peer dependencies on webpack with conflicting versions, this package is as-written incompatible with strict-peer-dependencies. This needs to have the dependencies on `heft-webpack4-plugin` and `heft-webpack5-plugin` be optional peer dependencies.

Alternatively it might make sense to do the opposite and have `heft-webpack4-plugin` and `heft-webpack5-plugin` each have their own version of the dev cert plugin and use an optional peer dependency on this plugin.

Reply:
Personally I think the latter would be a cleaner structure.

Reply:
>Alternatively it might make sense to do the opposite and have heft-webpack4-plugin and heft-webpack5-plugin each have their own version of the dev cert plugin and use an optional peer dependency on this plugin.

That seems like unnecessary code duplication. We could build only the "update the dev server config" functionality directly into those plugins and leave the trust/untrust functionality here.

Reply:
@iclanton which plugins use that functionality? Let's talk offline about this, since I don't have much of a handle on the use-case.

Reply:
After more discussion I definitely favor having each individual webpack plugin have a direct dependency on `@rushstack/debug-certificate-manager` and read the cert itself. This plugin is then only for trust/untrust commands and has no dependency on webpack, which makes the dependencies a lot cleaner.

Reply:
It also results in a lot less total code, since we no longer need to inspect webpack dev server versions.